ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml

```diff
@@ -0,0 +1,72 @@
+canonical_go_repository: github.com/openshift/tektoncd-pipeline-operator
+
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
```

Comment:
Remove the `cluster:` part, it's not necessary and makes ci-operator treat the source as an external cluster.

Reply:
@petr-muller in promotion too?